composer.json

```diff
@@ -30,5 +31,8 @@
     "autoload": {
```

Comment:
You also needs to change the suggest:

     "predis/predis": "If you want to use predis (currently only v0.8.x is supported).",